_raml/apps/traits.raml

```diff
@@ -1,7 +1,7 @@
 - locale:
     queryParameters:
       locale:
-        description: Represents the language code according to ISO 639-1 and the country code according to ISO 3166-1.
+        description: Represents the language code according to ISO 639-1 and the country code according to ISO 3166-1. We recommend to always set the required locale with the request header. If the default language had been changed in the administration, it might take up to 1 day until this change affects the shop.
```

Comment:
Two things: 
- we should mention what happens when this optional parameter is not used.
- it is a query parameter and not a request header

May suggestion:
If not provided the default language of the shop is used. 
Since it might take up to 1 day until this change affects the shop, if the default language had been changed in the administration, we suggest to always send this parameter with your request.